input/delimiter/fallback_delimiter.go

```diff
@@ -1,4 +1,4 @@
-package input
+package delimiter
```

Comment:
I changed the package name and renamed the `Delimiter` to be the `FallbackDelimiter`.